application/comit_node/src/swap_protocols/rfc003/ethereum/contract_templates/ether_contract.asm

```diff
@@ -2,6 +2,15 @@
     // Placeholder for deployment timestamp
     0x50000005
 
+    // Load expect secret size
+    0x6006
+
+    // Load received secret size
+    calldatasize
+
+    // Compare secret size
+    eq
+
     // Load secret into memory
     calldatacopy(0, 0, 32)
```

Comment:
🤡 the last param of `calldatacopy` is the amount of bytes which will be copied, i.e. 32 hardcoded. This should be the same as the value behind the placeholder `0x6006`. Otherwise we will likely end up in weird errors in the future. 
However, I'm not sure if this value is limited to `32` already.